R/periods.r

```diff
@@ -465,6 +476,12 @@ months.numeric <- function(x, abbreviate) {
   period(month = x)
 }
 
+.period_mapping = list(
```

Comment:
I think this makes sense but I'm not sure about R scoping rules and how long this object will live for..